server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequest.java

```diff
@@ -325,6 +325,16 @@ public static SortBy of(String value) {
             this(in.readString(), in.readString());
         }
 
+        public static After fromParam(String param) {
+            final String[] parts = param.split(",");
+            if (parts.length != 2) {
+                throw new IllegalArgumentException(
+                    "after param must be of the form ${sort_value},${snapshot_name} but was [" + param + "]"
```

Comment:
Should we not include the repo as a tiebreaker too? I.e., the format is `sort_value,repo_name,snapshot_name`?

Reply:
++ will do